jenkins/runlocal

```diff
@@ -1,19 +1,17 @@
 #!/bin/bash
 set -o errexit
 
-export PB_INTERNAL_CONTAINER_REG=images.paas.redhat.com
+export PB_CONTAINER_REG=images.paas.redhat.com
 export PB_ORG_NAME=pbench
 export PB_SERVER_IMAGE_NAME=pbench-server-ci
 export PB_SERVER_IMAGE_TAG=${USER}
-export PB_SERVER_IMAGE="${PB_INTERNAL_CONTAINER_REG}/${PB_ORG_NAME}/${PB_SERVER_IMAGE_NAME}"
-export PB_POD_NAME=pbench-in-a-can
 export PB_DASHBOARD_DIR=$(pwd)/dashboard/build
 
 make -C server/rpm clean rpm
 
-#buildah login -u='$app' -p="${__LOGIN_SECRET__}" ${PB_INTERNAL_CONTAINER_REG}
+#buildah login -u='$app' -p="${__LOGIN_SECRET__}" ${PB_CONTAINER_REG}
 
```

Comment:
Do we really want to be including commented-out code in this file?  (Wouldn't it be better to have a human-readable code comment explaining how to get the `-p` value for that `-u` value?)